GitHub.js

```diff
@@ -79,13 +80,24 @@ function doWeb(doc, url) {
 }
 
 
-function scrape(doc, _url) {
+function scrape(doc, url) {
 	var item = new Z.Item("computerProgram");
 	
 	var repo = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
 	
 	// basic metadata from the meta tags in the head
 	item.url = ZU.xpathText(doc, '//meta[@property="og:url"]/@content');
+	if (url.includes('/blob/') && !item.url.includes('/blob/')) {
+		// github is doing something weird with the og:url meta tag right now -
+		// it always points to the repo root (e.g. zotero/translators), even
+		// when we're in a specific directory/file (e.g. zotero/translators/
+		// blob/master/GitHub.js). this fix (hopefully) won't stick around
+		// long-term, but for now, let's just grab the user-facing permalink
+		let permalink = attr(doc, '.js-permalink-shortcut', 'href');
+		if (permalink) {
+			item.url = 'https://' + doc.location.host + permalink;
```

Comment:
`doc.location.host` → `github.com` — no reason not to hard-code, is there?

Reply:
Hmm, my reasoning was that it's one less thing to change if the translator is extended to support Enterprise installs / [an alternate domain](https://en.wikipedia.org/wiki/Censorship_of_GitHub) / some other weird case. But not *really*.

Reply:
I guess that's fine.

Maybe worth including the `clipboard-copy` element, which has the absolute URL in its `value` attribute, as a backup to `.js-permalink-shortcut`?